Source: me@gmail.com
Upstream: No
Reason: Fix desktop entry. (We changed the location of supertuxkart excutable)
--- supertuxkart-0.6.2/data/supertuxkart_desktop.template~	2009-09-16 01:39:25.593653064 +0800
+++ supertuxkart-0.6.2/data/supertuxkart_desktop.template	2009-09-16 01:41:21.178857934 +0800
@@ -1,17 +1,16 @@
-
 [Desktop Entry]
 Encoding=UTF-8
 Name=Supertuxkart
 Version=0.5
-Icon=PREFIX/share/pixmaps/supertuxkart_64.xpm
+Icon=supertuxkart_64
 Comment=A kart racing game
 Comment[fr_FR]=Un jeu de karting
 Comment[ro_RO]=Un joc de curse cu carturi
 Comment[de_DE]=Ein Kart-Rennspiel
-Exec=PREFIX/games/supertuxkart --log=file
+Exec=supertuxkart --log=file
 Terminal=false
 StartupNotify=false
 Type=Application
-TryExec=PREFIX/games/supertuxkart
+TryExec=supertuxkart
 Categories=Game;ArcadeGame
 GenericName=
